Vue-Basic-Video-Chat/js/app.js

```diff
@@ -0,0 +1,105 @@
+Vue.component('publisher', {
+  template: '<div></div>',
+  props: {
+    session: {
+      type: OT.Session,
+      required: false
+    },
+    opts: {
+      type: Object,
+      required: false
+    }
+  },
+  mounted: function() {
+    publisher = OT.initPublisher(this.$el, this.opts, (err) => {
+      if (err) {
+        this.$emit('error', err);
+      } else {
+        this.$emit('publisherCompleted');
+      }
+    });
+    this.$emit('publisherCreated', publisher);
+    if (this.session && this.session.isConnected()) {
+      this.session.publish(publisher);
+    } else {
+      this.session.on('sessionConnected', () => {
```

Comment:
Can this be `this.session.once`?

Reply:
I think it should be on. If you disconnect and connect again we will want to publish the publisher again.